### ISSUE TITLE
Error adding flows  e.z.replace is not a function

### ISSUE THREAD

**Issue by kisdaniel** [node-red/node-red#2774]
I have added https://github.com/ben423423n32j14e/zigbee2mqttadminpanel nodes to my node and after that I get the "Error adding flows  e.z.replace is not a function" message on screen.

![image](https://user-images.githubusercontent.com/16598258/101064704-966ba100-3594-11eb-9372-d2f988046b1c.png)

In javascript console the following message appears:

```
Uncaught TypeError: e.z.replace is not a function
    at red.min.js:16
    at Object.eachConfig (red.min.js:16)
    at Object.t [as refresh] (red.min.js:16)
    at Object.onchange (red.min.js:16)
    at R (red.min.js:16)
    at HTMLAnchorElement.D (red.min.js:16)
    at HTMLAnchorElement.dispatch (vendor.js:2)
    at HTMLAnchorElement.v.handle (vendor.js:2)
```

npm  version: 6.14.8
node version: v14.15.1
node-red version: 1.2.4  
node-red-contrib-influxdb version: 0.5.0
node-red-dashboard version 2.24.1-beta
npm version: 6.14.8

**Comment by knolleary** [comment]
I believe this was fixed in 1.2.5.

**Comment by knolleary** [comment]
Closing as I believe this was fixed in a later release from the one it was reported against, and there has been no follow up to say otherwise.
